src/app/components/Staking/FinalityProviders/FinalityProviders.tsx

```diff
@@ -6,8 +6,8 @@ import {
 } from "@/app/components/Loading/Loading";
 import { QueryMeta } from "@/app/types/api";
 import { FinalityProvider as FinalityProviderInterface } from "@/app/types/finalityProviders";
-import { getNetworkConfig } from "@/config/network.config";
-import { Network } from "@/utils/wallet/wallet_provider";
+// import { getNetworkConfig } from "@/config/network.config";
+// import { Network } from "@/utils/wallet/wallet_provider";
```

Comment:
No need to leave comments for now, we can remove them.

Reply:
Removed